.vscode/provardx-copyright.code-snippets

```diff
@@ -0,0 +1,15 @@
+{
+	"Insert Copyright Header": {
+		"prefix": "copyright",
+		"body": [
+		  "/*",
+		  " * Copyright (c) 2020 Make Positive Provar Ltd",
+		  " * All rights reserved.",
+		  " * Licensed under the BSD 3-Clause license.",
+		  " * For full license text, see LICENSE.md file in the repo root or https://opensource.org/licenses/BSD-3-Clause",
```

Comment:
Don't reference an external license page. Include our license in the project and make sure it's our company name at the top of it.

Reply:
We already have a LICENSE.md in our project with company name on it:
https://github.com/ProvarTesting/provardx/blob/development/LICENSE.md